execution/gethexec/blockchain.go

```diff
@@ -83,6 +94,7 @@ var TestCachingConfig = CachingConfig{
 	MaxNumberOfBlocksToSkipStateSaving: 0,
 	MaxAmountOfGasToSkipStateSaving:    0,
 	StylusLRUCache:                     0,
+	StateScheme:                        env.GetTestStateScheme(),
```

Comment:
I don't like calling GetTestStateScheme from here.
when not building gethexec tests, the gethexec module should optimally not have test-only dependencies.
Also - this is conceptually a constant (go doesn't support struct constants) and making it depend on run-time env variable is.. iffy

I think it's only used by system tests. Might be a good time to move ConfigDefaultNonSequencerTest and ConfigDefaultTest funcs into system tests and check the env variable from there.

Reply:
Done, moved test configs from gethexec to system_tests.